src/utils.rs

```diff
@@ -10,6 +10,69 @@
 
 use syntax::ast::Visibility;
 
+pub trait FindUncommented {
+    fn find_uncommented(&self, pat: &str) -> Option<usize>;
+}
+
+impl FindUncommented for str {
+    fn find_uncommented(&self, pat: &str) -> Option<usize> {
+        let needle = pat.as_bytes();
+        let needle_len = needle.len();
+        let mut match_count: usize = 0;
+        let mut possible_comment = false;
+
+        for (i, b) in self.as_bytes().iter().cloned().enumerate() {
+            if b == needle[match_count] {
+                match_count += 1;
+                if match_count == needle_len {
+                    return Some((i + 1) - needle_len);
+                }
+            } else {
+                match_count = 0;
+            }
+
+            if possible_comment {
+                if b == '/' as u8 {
```

Comment:
I think you should be iterating over chars, rather than bytes, it seems like you might get subtle errors if you hit a wide char where one of the bytes == `/`, or some such thing.